README.md

```diff
@@ -1,17 +1,48 @@
 # Xpense
 
-The Xpense project is an open source platform for housekeeping.
+The **Xpense** project is an open source platform for housekeeping.
```

Comment:
I would define it as "financial housekeeping", since you're not aiming for all topics related to housekeeping.